lib/dor/text_extraction/aws_provider.rb

```diff
@@ -23,11 +23,6 @@ def bucket_name
         Settings.aws.speech_to_text.base_s3_bucket
       end
 
-      # @return [String]
-      def sqs_todo_queue_url
-        Settings.aws.speech_to_text.sqs_todo_queue_url
-      end
-
```

Comment:
We no longer need to the todo SQS queue since AWS Batch has its own queue that it manages.